addons/point_of_sale/static/src/app/navbar/navbar.js

```diff
@@ -108,9 +122,6 @@ export class Navbar extends Component {
     showBackButton() {
         return this.pos.showBackButton();
     }
-    showBackButtonMobile() {
-        return this.pos.showBackButton() && this.ui.isSmall && !this.pos.scanning;
```

Comment:
You should maybe keep the hiding of the backbutton if the scanning is active ?

Reply:
in mobile in normal pos if scanning is active you are on the productslist view, so you don't have a back button anyways and in restaurant you will indeed have the back button when scanning is active, and clicking it will take you to the floors page, but i think that is a good behavior. What do you think ?

Reply:
Ok I did not know how the barcode scanner was working and that it was only working in the product screen, sorry my bad